addons/calendar/models/calendar_event.py

```diff
@@ -629,19 +629,15 @@ def write(self, values):
 
         current_attendees = self.filtered('active').attendee_ids
         if 'partner_ids' in values:
-            # we send to all partners and not only the new ones
-            (current_attendees - previous_attendees)._send_mail_to_attendees(
-                self.env.ref('calendar.calendar_template_meeting_invitation', raise_if_not_found=False)
-            )
+            self._send_template_to_attendees('calendar.calendar_template_meeting_invitation', attendees=current_attendees - previous_attendees)
```

Comment:
:socks: To reduce a bit the line length perhaps this
```python
attendees_to_notify = current_attendees - previous_attendees
self._send_template_to_attendees('calendar.calendar_template_meeting_invitation', attendees=attendees_to_notify)
```
or just in multi-lines, the choice is yours :smile: